typescript/cli/src/commands/core.ts

```diff
@@ -146,7 +146,7 @@ export const read: CommandModuleWithContext<{
       const coreConfig = await evmCoreReader.deriveCoreConfig(mailbox);
       writeYamlOrJson(configFilePath, coreConfig, 'yaml');
       logGreen(`✅ Core config written successfully to ${configFilePath}:\n`);
-      log(indentYamlOrJson(yamlStringify(coreConfig, null, 2), 4));
+      logYamlIfUnderMaxLines(coreConfig, MAX_READ_LINE_OUTPUT);
```

Comment:
with the below update, just
```suggestion
      logYamlIfUnderMaxLines(coreConfig);
```